### ISSUE TITLE
Aeon WallMote Quad ZW130 Swipe

### ISSUE THREAD

**Issue by Sian-Lee-SA** [AlCalzone/node-zwave-js#5399]
I did a quick search regarding this device but there was no issue relating to the swipe function of the device. Previosuly OpenZWave was able to intercept this action but a custom script was needed to make better use of the function.

I can not find any event related to the swipe down or swipe up action for this device with Z-Wave JS. The config also has this option enabled (AKA button slide).

**Comment by AlCalzone** [comment]
You have to add associations for this feature:
![grafik](https://user-images.githubusercontent.com/17641229/216366565-23b7001d-2d78-46f1-9d7f-03b44aa509ec.png)
Each button needs its own association to node 1:
![grafik](https://user-images.githubusercontent.com/17641229/216366708-2001ab37-f6e9-4753-b180-dd5c2bef72a9.png)

If you do that, you get Start/Stop Level Change notifications:
![grafik](https://user-images.githubusercontent.com/17641229/216367013-bc8597e1-d694-4d8e-96e3-7fe4521bba25.png)
I'm not sure how they are exposed in HA though.

**Comment by kpine** [comment]
> I'm not sure how they are exposed in HA though.

Via Notification events. https://www.home-assistant.io/integrations/zwave_js/#multilevel-switch-command-class

**Comment by Sian-Lee-SA** [comment]
Thanks, That works except theres just one problem, I can't differentiate if the swipe was from the left or right side.

**Comment by AlCalzone** [comment]
You're right. We don't pass the endpoint information with those notifications.

When we do, you need to set up a multi channel association for the buttons:
![image](https://user-images.githubusercontent.com/17641229/216444391-e9893282-9890-4b25-b2a3-401adedde0a6.png)

**Comment by Sian-Lee-SA** [comment]
Ok, at first there was nothing in the endpoint 1-4. So I figured that I would reinterview the node (because why not?). All of a sudden I had 4 entries added automatically in the group for the Dimmer key (endpoints 1-4). I then did a test and I was getting duplicate notifications. So I removed those multileveled buttons and tested.

I can see the event like from the button (multilevel) was doing but nothing through Home Assistant anymore unfortunately. Sorry, I'm more experienced with Zigbee than Z-Wave and these things are similar but not everything will be clear right away so excuse my misunderstandings.

![Screenshot 2023-02-03 093135](https://user-images.githubusercontent.com/11685433/216471189-fa4c777d-ee50-4a87-8b47-90b0136acde4.png)

![Screenshot 2023-02-03 093209](https://user-images.githubusercontent.com/11685433/216471217-3b768614-9787-41b6-bf31-95ae9ee52bc2.png)


Update: Even though I can't get it through HA, I can get it from MQTT with Node-RED which would suffice but still not seeing anything to differentiate the left and right. Just the time and if it's up or down, no endpoint info etc.

SORRY just read your comment again and notice you state "When we do" so I am assuming this will be on the roadmap and a future update will allow me to get that endpoint that called the dimming? Apologise for the rundown in this comment but will leave it for reference.
